Samples/auth/Office-Add-in-Microsoft-Graph-React/README.md

```diff
@@ -100,6 +97,7 @@ Version  | Date | Comments
 ### Configure the sample
```

Comment:
I tested on v20.12.2 (LTS) and works fine. However older versions than 18 can fail so I updated the readme with correct version info.

Reply:
Node LTS v20 works. You might need to turn off engines-strict in project .npmrc to avoid error due to engines version specification for packages that say they don’t work with Node v20 but they actually do.